.github/workflows/upload_app_store.yml

```diff
@@ -0,0 +1,62 @@
+name: Upload IPA to App Store
+on: workflow_dispatch
+jobs:
+  check_branch:
+    name: "Check branch ('release*' or 'main')."
+    runs-on: macos-11
+    steps:
+      - id: early_exit
+        if: ${{ github.ref != 'refs/heads/main' && startsWith(github.ref, 'refs/heads/release') != true }}
+        name: Early exit failure if not main or release branch.
+        run: exit 1
+  build_test_upload:
+    name: Build, test, and upload to App Store
+    needs: check_branch
+    runs-on: macos-11
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+            node-version: 16
+            cache: "npm"
+            cache-dependency-path: "**/package-lock.json"
+      - name: Build core and features
+        run: ./build.js
+        env:
+            POLYPOD_POLYPEDIA_REPORT_URL: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_URL }}
+            POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION }}
+      - name: Lock xcode version
+        run: sudo xcode-select -s "/Applications/Xcode_13.2.1.app"
+      - name: Run tests
+        working-directory: ios
+        run: |
+          set -o pipefail
+          make test | xcpretty
```

Comment:
Have you run this? Unless you've change the destination somewhere and I'm not aware of it, [the combination Xcode-OS will not have our targets available](https://github.com/polypoly-eu/polyPod/runs/5004831598?check_suite_focus=true)

Reply:
Yes, we need to upgrade to 15.2 target. I have reverted to 14.4 until the merge, or testing yaml is updated to 15.2.